### ISSUE TITLE
Is high-gamma-dataset  a motor imagery dataset or a motor execution dataset?

### ISSUE THREAD

**Issue by sayaiiii** [robintibor/high-gamma-dataset#6]
I think it's a motor execution dataset after reading sesseion A.6 of https://arxiv.org/abs/1703.05051, but I'm not sure about that.
Could you please tell me if it is a motor execution dataset？ Thanks.

**Comment by robintibor** [comment]
Yes you are right it is motor execution dataset as specified in the supplementary section A6 at https://onlinelibrary.wiley.com/action/downloadSupplement?doi=10.1002%2Fhbm.23730&file=hbm23730-sup-0001-suppinfo.pdf Supplementary Information (scroll to bottom)
